demisto_sdk/commands/run_playbook/playbook_runner.py

```diff
@@ -45,112 +41,60 @@ def run_playbook(self) -> int:
         """
         # create an incident with the given playbook
         try:
-            incident_id = self.create_incident_with_playbook(
-                incident_name=f"inc_{self.playbook_id}", playbook_id=self.playbook_id
+            incident = self.instance_api_client.create_incident(
+                name=f"incident-test-{self.playbook_id}-playbook",
+                attached_playbook_id=self.playbook_id,
+            )
+        except ApiException as error:
+            logger.error(
+                f"Could not create incident in {self.instance_api_client.base_url} to test playbook {self.playbook_id}, error\n{error})"
             )
-        except ApiException as a:
-            logger.info(f"[red]{a}[/red]")
             return 1
 
-        work_plan_link = self.base_link_to_workplan + str(incident_id)
+        logger.info(
+            f"[green]The playbook: {self.playbook_id} was triggered successfully.[/green]"
+        )
+
+        incident_id = incident.id
+        work_plan_link = self.instance_api_client.get_incident_work_plan_url(
+            incident_id
+        )
+
         if self.should_wait:
             logger.info(
-                f"[green]Waiting for the playbook to finish running.. \n"
-                f"To see the playbook run in real-time please go to : {work_plan_link}[/green]"
+                f"Waiting for the playbook to finish running.. \n"
+                f"To see the playbook run in real-time visit: {work_plan_link}"
             )
+            try:
+                playbook_state_raw_response = (
+                    self.instance_api_client.poll_playbook_state(
+                        incident_id,
+                        expected_states={
```

Comment:
WAITING means waiting on manual input no? This isnt a status that should change. should we stop on this too?

Reply:
the idea here is not to run a playbook that waits for any manual input, in case it would be stack on "WAITING" an exception would be raised indicating that the playbook is stuck on waiting.